packages/SwingSet/src/devices/plugin/device-plugin.js

```diff
@@ -115,7 +115,7 @@ export function buildRootDeviceNode(tools) {
   function send(index, obj) {
     const mod = connectedMods[index];
     // console.info('send', index, obj, mod);
-    assert(mod, X`No module associated with ${index}`, TypeError);
+    mod || Fail`No module associated with ${index}`;
```

Comment:
We should probably maintain this and similar use of `assert`'s third argument so the PR doesn't affect input/output behavior.
```suggestion
    mod || assert.fail(X`No module associated with ${index}`, TypeError);
```

Reply:
All those `TypeErrors` were historical baggage, and this just gets rid of them.